scripts/python/update-user-defined-dimension-attribute-type.py

```
@@ -0,0 +1,44 @@
+from django.db.models import Q
```

Ruff

`django.db.models.Q` imported but unused

Remove unused import: `django.db.models.Q`

(F401)